cmd/device/update/update.go

```diff
@@ -69,32 +68,32 @@ func deviceHandler(cmd *cobra.Command, args []string) error {
 	}
 
 	if file != "" {
-		return updateDevicesFromFile()
+		return updateDevicesFromFile(cmd)
```

Comment:
why "cmd" instead of just the "context" in the cmd? You do not seem to be using anywhere the cmd .. unnecessarily constraining your use of these functions to  cobra command. your true goal is to access the context to be able to send signals such as abort etc.